<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <exportmacro>PLUGIN_CAGE_3D_DEFORMATION_EXPORT</exportmacro>
 <class>Cage3dDeformation_Dialog</class>
 <widget class="QDialog" name="Cage3dDeformation_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cage 3d Deformation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Deformed map :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListWidget" name="list_deformed_maps"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Position attribute :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="combo_deformedPositionAttribute">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Control map :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QListWidget" name="list_control_maps"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Position attribute :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="combo_controlPositionAttribute">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="button_linkUnlink">
     <property name="text">
      <string>Link</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_ok</sender>
   <signal>clicked()</signal>
   <receiver>Cage3dDeformation_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Cage3dDeformation_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
